enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,1319 @@
+    reason: WantedByEndpointPublishingStrategy
+    status: "True"
+    type: LoadBalancerManaged
+  - lastTransitionTime: "2024-04-09T22:35:16Z"
+    message: The LoadBalancer service is provisioned
+    reason: LoadBalancerProvisioned
+    status: "True"
+    type: LoadBalancerReady
+  - lastTransitionTime: "2024-04-09T22:35:12Z"
+    message: LoadBalancer is not progressing
+    reason: LoadBalancerNotProgressing
+    status: "False"
+    type: LoadBalancerProgressing
+  - lastTransitionTime: "2024-04-09T22:35:12Z"
+    message: DNS management is supported and zones are specified in the cluster DNS
+      config.
+    reason: Normal
+    status: "True"
+    type: DNSManaged
+  - lastTransitionTime: "2024-04-09T22:35:28Z"
+    message: The record is provisioned in all reported zones.
+    reason: NoFailedZones
+    status: "True"
+    type: DNSReady
+  - lastTransitionTime: "2024-04-09T22:42:57Z"
+    status: "True"
+    type: Available
+  - lastTransitionTime: "2024-04-09T22:43:27Z"
+    status: "False"
+    type: Progressing
+  - lastTransitionTime: "2024-04-09T22:43:22Z"
+    status: "False"
+    type: Degraded
+  - lastTransitionTime: "2024-04-09T22:35:12Z"
+    message: IngressController is upgradeable.
+    reason: Upgradeable
+    status: "True"
+    type: Upgradeable
+  - lastTransitionTime: "2024-04-09T22:35:12Z"
+    message: No evaluation condition is detected.
+    reason: NoEvaluationCondition
+    status: "False"
+    type: EvaluationConditionsDetected
+  - lastTransitionTime: "2024-04-09T22:43:21Z"
+    message: Canary route checks for the default ingress controller are successful
+    reason: CanaryChecksSucceeding
+    status: "True"
+    type: CanaryChecksSucceeding
+  domain: apps.misalunkeip16.devcluster.openshift.com
+  endpointPublishingStrategy:
+    loadBalancer:
+      dnsManagementPolicy: Managed
+      providerParameters:
+        aws:
+          classicLoadBalancer:
+            connectionIdleTimeout: 0s
+          type: Classic
+        type: AWS
+      scope: External
+    type: LoadBalancerService
+  observedGeneration: 1
+  selector: ingresscontroller.operator.openshift.io/deployment-ingresscontroller=default
+  tlsProfile:
+    ciphers:
+    - ECDHE-ECDSA-AES128-GCM-SHA256
+    - ECDHE-RSA-AES128-GCM-SHA256
+    - ECDHE-ECDSA-AES256-GCM-SHA384
+    - ECDHE-RSA-AES256-GCM-SHA384
+    - ECDHE-ECDSA-CHACHA20-POLY1305
+    - ECDHE-RSA-CHACHA20-POLY1305
+    - DHE-RSA-AES128-GCM-SHA256
+    - DHE-RSA-AES256-GCM-SHA384
+    - TLS_AES_128_GCM_SHA256
+    - TLS_AES_256_GCM_SHA384
+    - TLS_CHACHA20_POLY1305_SHA256
+    minTLSVersion: VersionTLS12
+```
+
+Add the `ingress.operator.openshift.io/auto-delete-load-balancer:` to the ingress controller.
+```sh
+% oc -n openshift-ingress-operator annotate ingresscontrollers/default ingress.operator.openshift.io/auto-delete-load-balancer=
+ingresscontroller.operator.openshift.io/default annotated
+```
```

Comment:
Can't they add this annotation at the same time that they add the EIP allocations?

Reply:
They can using oc edit but I am showing it through oc commands.